modin/core/storage_formats/base/query_compiler.py

```diff
@@ -1118,7 +1118,7 @@ def merge_asof(
         tolerance=None,
         allow_exact_matches: bool = True,
         direction: str = "backward",
-    ):
+    ):  # noqa: GL08
```

Comment:
why?

Reply:
Previously, an empty string was assigned and the problem was not visible. 

Now the question arises: why did the checker not pay attention to the empty string? This is most likely another bug :) In this situation, the successor does not override this function (`PandasQueryCompiler.merge_asof is BaseQueryCompiler.merge_asof == True`), it turns out that we inherit an empty string and assign `__doc_inherited__` label. And since the function is the same, we essentially disabled the check not only for the function that is docstring inherited, but also the function from which we took it.

Reply:
Thanks for the explanation :)